Description: Removes a left-over print that exposes account data.
Bug: https://sourceforge.net/p/ddclient/bugs/72/
Origin: https://sourceforge.net/p/ddclient/code/166/
Author: wimpunk <wimpunk+github@gmail.com>
Reviewed-by: Torsten Landschoff <torsten@debian.org>
Applied-Upstream: r166

--- a/ddclient
+++ b/ddclient
@@ -2588,10 +2588,6 @@ sub nic_noip_update {
 	$url  .= "&myip=";
 	$url  .= $ip            if $ip;
 
-
-	print "here..." . $config{$h}{'login'} . " --> " . $config{$h}{'password'} . "\n";
-	
-
 	my $reply = geturl(opt('proxy'), $url, $config{$h}{'login'}, $config{$h}{'password'});
 	if (!defined($reply) || !$reply) {
 	    failed("updating %s: Could not connect to %s.", $hosts, $config{$h}{'server'});
